unit-tests/py/rspy/devices.py

```diff
@@ -2,18 +2,27 @@
 # Copyright(c) 2021 Intel Corporation. All Rights Reserved.
 
 from rspy import log
+import sys, os
 
 # We need both pyrealsense2 and acroname. We can work without acroname, but
 # without rs no devices at all will be returned.
 try:
     import pyrealsense2 as rs
     log.d( rs )
+    #
+    # Have to add site-packages, just in case: if -S was used, or parent script played with
+    # sys.path (as run-unit-tests does), then we may not have it!
+    sys.path += [os.path.join( os.path.dirname( sys.executable ), 'lib', 'site-packages')]
+    #
     try:
         from rspy import acroname
     except ModuleNotFoundError:
         # Error should have already been printed
         # We assume there's no brainstem library, meaning no acroname either
+        log.d( 'sys.path=', sys.path )
         acroname = None
+    #
+    sys.path = sys.path[:-1]  # remove what we added
```

Comment:
If we want to use a third party lib like numpy in our test it will not work right?
If yes, than we need to think of another way to solve this.

Reply:
Right, it won't work (unless numpy isn't in site-packages) unless the module is in lib/.
One thing I can do is remove site-packages before loading librealsense, but it's hard to control when a script imports pyrs.
For now, it'll work.